src/main/fc/rc_adjustments.c

```diff
@@ -226,6 +226,10 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
         .adjustmentFunction = ADJUSTMENT_LED_PROFILE,
         .mode = ADJUSTMENT_MODE_SELECT,
         .data = { .switchPositions = 3 }
+    }, {
+        .adjustmentFunction = ADJUSTMENT_LED_DIMMER,
+        .mode = ADJUSTMENT_MODE_SELECT,
```

Comment:
Isn't `ADJUSTMENT_MODE_STEP` designed for this situation?

Reply:
The idea at least to me is to use a pot or slider as a simple way to control LED brightness, a step adjustment would work but seems like overkill